File: front/lib/domain/product.dart
class Product {
  final int pk;
  final String image;
  final String name;
  final double price;
  final String description;

  Product(
      {
        required this.pk,
        required String image,
      required this.name,
      required this.price,
      required this.description})
      : image = image.replaceFirst(RegExp(r"localhost"), "10.0.2.2");
}

File: back/src/models/product.py
from django.db import models
from .user import User


class Product(models.Model):
    user = models.ForeignKey(User, on_delete=models.CASCADE)
    name = models.CharField(blank=False, max_length=200)
    description = models.TextField(blank=False)
    image = models.CharField(blank=False, max_length=200)
    price = models.FloatField()
